src/App.js

```diff
@@ -1,25 +1,45 @@
-import logo from './logo.svg';
-import './App.css';
+// import logo from './logo.svg';
+// import './App.css';
+import React, {Component} from 'react';
+import axios from 'axios';
 
-function App() {
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state={
+      searchQuery: '',
+      location: {},
+      error: false,
+    }
+  }
+  
+  // getLocation = async () +> {
+  //   const url = ''
+  // }
+render(){
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload. Changes
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+
+    <div> 
+    <input onChange={(event) => this. setState({searchQuery: event.target.value})} placeholder="Search"></input>
+    <button onClick={this.getLocation}>Explore!</button>
```

Comment:
Really nice work getting this going! 👍 

Feedback here: Remove `setState()` call in render as this can cause infinite loops and in general can make component less maintainable. If you do a quick search, you can read more about this. JB and Sara also touched on why  we want to avoid calling setState here :) 

The render function should remain pure (sometimes setState is done in render, but for most cases, we want to avoid it).